Source/Scene/Model.js

```diff
@@ -772,6 +781,10 @@ define([
         return JSON.parse(json);
     }
 
+    Model.getDefaultCacheKey = function(url) {
```

Comment:
Ah, I missed that.  I think it is fine; I'll let you know if I come up with anything else.

In the meantime, explicitly mark it `@private`.